.github/dependabot.yaml

```diff
@@ -0,0 +1,20 @@
+# THIS FILE IS AUTOMATICALLY GENERATED. DO NOT EDIT.
+# Edit https://github.com/cert-manager/makefile-modules/blob/main/modules/repository-base/base/.github/dependabot.yaml instead.
```

Comment:
Was **this** file automatically generated? If so, how?
Please document the process in the README file.

Reply:
No, this file is not auto-generate, but it will be copied into the project & that copy will be auto-generated.

Reply:
Indeed, my mistake. @wallrj

Reply:
This file is for updating the dependencies in *this* project, right? e.g.
* https://github.com/cert-manager/makefile-modules/blob/5f23abc93e192e49875d3e7de88ddff00c7331a8/modules/oci-image/image_tool/go.mod#L5-L8

So the comment in this file that says "THIS FILE IS AUTOMATICALLY GENERATED. DO NOT EDIT" is wrong and should be removed.